### DISCUSSION TITLE
Add Couchbase product profile

### DISCUSSION THREAD

**Discussion by 1tylermitchell** [superlinked/VectorHub#378] [VDB Comparison]
Hi all, I just put out the PR for a Couchbase product profile. It is a major NoSQL database with vector search capabilities, among others, and key to a bunch of global industries and companies. I hope I didn't miss anything in the process, let me know if I can improve anything for you to make it easier.

https://github.com/superlinked/VectorHub/pull/377

**Comment by 1tylermitchell** [comment]
Another light update is available in PR #437 

Also wondering if vector compression approaches are represented in the comparison matrix, I couldn't see any but thought I'd ask.
Thanks!

**Reply by svonava** [reply]
Hi @1tylermitchell - thanks for the update! Are you refering to quantization/built-in dimensionality reduction? 

There already are int8 and binary quantization support columns:
<img width="942" alt="image" src="https://github.com/user-attachments/assets/7f01cec3-e721-41e8-8a2f-7a74fb1ad42e">
